src/templates/admin/core/active_submissions.html

```diff
@@ -31,14 +31,12 @@ <h2>Submission List</h2>
                     </div>
                 </div>
             </div>
-            <div class="large-4 columns">
-                <div class="box">
-
-                    <div class="content">
-
-                        <div class="row">
-                            <form id="filter_form" class="autoSubmit" method="POST">
-                                {% if editor %}
+            {% if editor %}
+                <div class="large-4 columns">
```

Comment:
With this 4 column div dissappearing, the row only has 8 columns declared, which can mess up the grid when introducing other rows or on other port sizes. It is worth setting the sibling to `large-12` when this div is not rendered